src/tag/serializers.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework.fields import SerializerMethodField
+from rest_framework.serializers import ModelSerializer
+
+from researchhub.serializers import DynamicModelFieldSerializer
+from tag.models import Concept
+
+
+class SimpleConceptSerializer(ModelSerializer):
+    class Meta:
+        model = Concept
+        fields = "__all__"
+
+class DynamicConceptSerializer(DynamicModelFieldSerializer):
+
+    openalex_id = SerializerMethodField()
+    display_name = SerializerMethodField()
+    description = SerializerMethodField()
```

Comment:
We don't need to define these method fields because they aren't a relational mapping to another table.

Reply:
done